### ISSUE TITLE
Proxy usage

### ISSUE THREAD

**Issue by BercziSandor** [nidhaloff/deep-translator#134]
* deep_translator version:  1.7.0
* Python version: 3.10
* Operating System: Windows 10 

### Description

Unable to import deep_translator in case of proxy needed.
It is not possible to define proxy before imports. 
Idea: Maybe the internal use of environment variebles http_proxy/https_proxy?

### What I Did

Code:
```
from deep_translator import GoogleTranslator
```

Output:
```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\A349312\git\priv\subtitle_translator\venv\lib\site-packages\requests\adapters.py", line 440, in send
    resp = conn.urlopen(
  File "C:\Users\A349312\git\priv\subtitle_translator\venv\lib\site-packages\urllib3\connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "C:\Users\A349312\git\priv\subtitle_translator\venv\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.cognitive.microsofttranslator.com', port=443): Max retries exceeded with url: /languages?api-version=3.0&scope=translation (Caused by ProxyError('Cannot connect to proxy.', OSError('Tunnel connection failed: 407 Proxy Authentication Required')))
```

**Comment by nidhaloff** [comment]
@BercziSandor thanks for opening the issue. I think this is caused by the get request to get supported languages by the ms translator. I will check it out asap

**Comment by nidhaloff** [comment]
should be fixed in v1.8.0

https://github.com/nidhaloff/deep-translator/pull/139
